contributors.yml

```diff
@@ -279,3 +279,4 @@
 - youngvform
 - zachdtaylor
 - zainfathoni
+- sidv1905
```

Comment:
We're trying to keep this sorted alphabetically, could you move it up ? ⬆️

Reply:
Done